README.md

```diff
@@ -288,6 +289,134 @@ Example changing the spec and triggering the build via make (feel free to automa
 $ make clean && VERSION=YOURVERSIONHERE make rpm
 ```
 
+## QM sub-package KVM
+
+The QM sub-package KVM includes drop-in configuration that enables the integration of Kernel-based Virtual Machine (KVM) management into the QM (Quality Management) container environment. This configuration allows users to easily configure and manage KVM virtual machines within the QM system, streamlining virtualization tasks in containerized setups.
+
+Below example step by step:
+
+Step 1:  clone QM repo, install libvirt packages, prepare some files inside QM and start the libvirt daemon.
+
+```bash
+$host> git clone https://github.com/containers/qm.git && cd qm
+$host> make qm_dropin_mount_bind_kvm
+$host> sudo dnf install rpmbuild/RPMS/noarch/qm_mount_bind_kvm-0.6.7-1.fc40.noarch.rpm
+$host> sudo podman restart qm  # if you have qm already running
+$host> sudo dnf --installroot /usr/lib/qm/rootfs/ install virt-install libvirt-daemon libvirt-daemon-qemu libvirt-daemon-kvm -y
+
+# Copy default network settings to /root dir inside QM (/usr/lib/qm/rootfs/root)
+$host> sudo cp /usr/share/libvirt/networks/default.xml /usr/lib/qm/rootfs/root
+
+Step 2: Preparing cloudinit files inside QM (/usr/lib/qm/rootfs/root)
+
+# Cloud-init files
```

Comment:
Why are you using cloud-init? i assume it is for the guest? 
Is it only for the password?  

I prefer to use this command after that one 
```
virt-customize --uninstall cloud-init --root-password password:Aa1234 -a ./<fedora-image>.qcow2
```

https://github.com/containers/qm/pull/608/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R321

Reply:
sounds a good plan.  If we merge can you test and improve it for us?